ignite/config/plugins/config.go

```diff
@@ -36,6 +36,9 @@ type Plugin struct {
 	Path string `yaml:"path"`
 	// With holds arguments passed to the plugin interface
 	With map[string]string `yaml:"with,omitempty"`
+	// Global holds whether the plugin is installed globally
+	// (default: $HOME/.ignite/plugins/plugins.yml) or locally for a chain.
+	Global bool `yaml:"global,omitempty"`
```

Comment:
I don't get the usage of this new field, it's not read anywhere.

Reply:
It's read [here](https://github.com/ignite/cli/blob/36db41933fcfde6ecf637138756c308f26562e65/ignite/cmd/plugin.go#L624) so we can print info about where the plugin is installed with the `list` command.

It's not super important, but it might be useful info

Reply:
Oh OK I didn't see there was a function. In that case, there's no need to include that field the yaml file, let's change the field tag to `yaml:"-"` OK ?